diff --git a/ios/ExpoNotifications/Notifications/DateComponentsSerializer.swift b/ios/ExpoNotifications/Notifications/DateComponentsSerializer.swift
index 3d70480183d3491d6f432154be2b11575eab5b3a..d6499cf046f4987cdcfe9f270d887fc925d698ac 100644
--- a/ios/ExpoNotifications/Notifications/DateComponentsSerializer.swift
+++ b/ios/ExpoNotifications/Notifications/DateComponentsSerializer.swift
@@ -5,9 +5,6 @@ class DateComponentsSerializer {
     serializedComponents["calendar"] = dateComponents.calendar?.identifier ?? NSNull()
     serializedComponents["timeZone"] = dateComponents.timeZone?.identifier ?? NSNull()
     serializedComponents["isLeapMonth"] = dateComponents.isLeapMonth ?? false
-    if #available(iOS 26.0, *) {
-      serializedComponents["isRepeatedDay"] = dateComponents.isRepeatedDay ?? false
-    }
 
 
     let map = calendarUnitsConversionMap()
